_docs/tasks/telemetry/metrics-logs.md

```diff
@@ -167,6 +167,9 @@ as the example application throughout this task.
    double_request_count{destination="reviews.default.svc.cluster.local",instance="istio-mixer.istio-system:42422",job="istio-mesh",message="twice the fun!",source="productpage.default.svc.cluster.local"}	2
    ```
 
+   For more on querying Prometheus for metric values, see the [Querying Istio
+   Metrics]({{home}}/docs/tasks/telemetry/querying-metrics.html) Task.
+
```

Comment:
move to last section and call it "Further reading".. thats what I am doing in all the other tasks.